plugins/module_utils/common/xml_to_json.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+"""
+The index_of plugin common code
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import xmltodict
```

Comment:
Import this library if engine option is set to `xmltodict` Refer https://github.com/ansible-collections/ansible.utils/blob/main/plugins/sub_plugins/cli_parser/xml_parser.py#L46

Reply:
ack